screen.js

```diff
@@ -27,4 +131,9 @@ class Screen {
   returnToPreviousScreen() {
     return this._returnToPreviousScreen;
   }
+
+  /** Resize the background when the window is resized. */
+  windowResized() {
```

Comment:
Who calls this function?  Don't you need to override the global `windowResized` function to handle the event?

Reply:
`windowResized()` in `main.js` calls `windowResized()` for the current game mode. I modified `windowResized()` to resize the canvas in addition to updating the background (`formatBackground()` checks the canvas size so resizing only the window and calling it does nothing).